src/Plugin/Field/FieldFormatter/TypedRelationFormatter.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Drupal\controlled_access_terms\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\Plugin\Field\FieldFormatter\EntityReferenceLabelFormatter;
+use Drupal\Core\Field\FieldItemListInterface;
+
+/**
+ * Plugin implementation of the 'TypedRelationFormatter'.
+ *
+ * @FieldFormatter(
+ *   id = "typed_relation_default",
+ *   label = @Translation("Typed Relation Formatter"),
+ *   field_types = {
+ *     "typed_relation"
+ *   }
+ * )
+ */
+class TypedRelationFormatter extends EntityReferenceLabelFormatter {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    $elements = parent::viewElements($items, $langcode);
+    $values = $items->getValue();
```

Comment:
Variable never used, can be removed.

Reply:
Ah! Yes, that was hold over from a previous strategy.